crates/holochain/tests/tests/sharded_gossip/mod.rs

```diff
@@ -74,6 +74,63 @@ impl From<TestConfig> for SweetConductorConfig {
     }
 }
 
+#[cfg(feature = "test_utils")]
+#[tokio::test(flavor = "multi_thread")]
+#[cfg_attr(target_os = "macos", ignore = "flaky")]
```

Comment:
It's troubling that this simple test is flaky on macos.

Reply:
I don't know that it is, but it's largely coppied from another test in this file. My experience is anything with timing (i.e. await_consistency) is flaky on mac because they are so slow.

Reply:
I don't think that's a fair assumption, we have lots and lots of tests with await_consistency that don't have the flaky macos flag. Including every test that creates a SweetConductorBatch, [which has an await_consistency built-in](https://github.com/holochain/holochain/blob/4639097b863f3dc35a5b5df1d1cf67d8ef9e4ede/crates/holochain/src/sweettest/sweet_conductor_batch.rs#L84)

Reply:
Unless it's actually the case that every test with await_consistency has a low but similar chance of flaking, and we just happened to mark as macos-flaky the ones that failed in CI once, but can't afford to mark them all as flaky? I assume that some are definitely flakier than others.

Reply:
What about creating an issue to investigate the macos flakiness of tests in this module?

Reply:
Shouldn't we investigate *all* flaky tests, though? I'm not sure we want to write separate issues for them all.

Reply:
Generally yes, but I think sharded gossip will become important very soon.